qemu/tests/cfg/virtio_scsi_mq.cfg

```diff
@@ -10,6 +10,9 @@
     system_image_drive_format = virtio
     bootindex_image1 = 0
     irq_regex = "virtio\d+-request"
+    RHEL.8:
+        # No need to pin interrupts to vcpus from bug 1609148's comments.
+        pin_interrupts2vcpus = no
```

Comment:
As we discussed, check whether interrupts have pinned to vcpus. If not, pin them.